src/octod/api/authorizations.d

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+
+    Provides wrappers on top of some methods documented in
+    https://developer.github.com/v3/oauth_authorizations/
+
+    Copyright: Copyright (c) 2016 Sociomantic Labs GmbH. All rights reserved
```

Comment:
Maybe is a good opportunity to start using the correct case for the copyright :D (sociomantic labs GmbH).